supergloo/docs/mesh/gloo-istio-mtls.md

```diff
@@ -0,0 +1,214 @@
+default            details-v1-68c7c8666d-wsbqc               2/2     Running     0          9s
+default            productpage-v1-54d799c966-jbmpt           2/2     Running     0          9s
+default            ratings-v1-8558d4458d-gk976               2/2     Running     0          9s
+default            reviews-v1-cb8655c75-pgkzv                2/2     Running     0          9s
+default            reviews-v2-7fc9bb6dcf-mdzjg               2/2     Running     0          9s
+default            reviews-v3-c995979bc-pp4ms                2/2     Running     0          9s
+istio-system       istio-citadel-6f444d9999-v6vcz            1/1     Running     0          28m
+istio-system       istio-cleanup-secrets-ncvx8               0/1     Completed   0          28m
+istio-system       istio-galley-685bb48846-nxbfs             1/1     Running     0          28m
+istio-system       istio-pilot-7959f4df76-gsbf5              2/2     Running     0          28m
+istio-system       istio-policy-66ccc8df5f-7b8t9             2/2     Running     0          28m
+istio-system       istio-security-post-install-9n5mb         0/1     Completed   0          28m
+istio-system       istio-sidecar-injector-5d8dd9448d-wdq9z   1/1     Running     0          28m
+istio-system       istio-telemetry-586ccd6d57-kdjnt          2/2     Running     0          28m
+supergloo-system   discovery-7bbbf86b66-vw2g5                1/1     Running     0          29m
+supergloo-system   supergloo-975bfbfb7-26m2f                 1/1     Running     0          29m
+```
+If the list includes the following pods then everything should be good to go.
+
+## Installing Gloo with supergloo
+
+Once the mandatory prerequisites have been completed, the installation of gloo using supergloo is a single step process. Assuming that the exact names of the earlier tutorials were used, the command will look as follows. 
+
+#### Option 1: CLI
+```bash
+supergloo install gloo --name gloo --target-meshes supergloo-system.my-istio
+```
+
+#### Option 2: yaml
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: supergloo.solo.io/v1
+kind: Install
+metadata:
+    name: gloo
+    namespace: supergloo-system
+spec:
+    ingress:
+        gloo:
+        glooVersion: 0.13.5
+        meshes:
+        - name: my-istio
+            namespace: supergloo-system
+    installationNamespace: gloo-system
+EOF
+```
+
+This command should add the following pods
+```bash
+gloo-system        gateway-9b648f4d-4bnlv                    1/1     Running     0          16s
+gloo-system        gateway-proxy-ddf675bc9-sw756             1/1     Running     0          15s
+gloo-system        gloo-8576cf6786-t455l                     1/1     Running     0          16s
+```
+
+The next step to routing mtls enabled traffic through gloo, is to decide on an upstream to route the traffic to.
+In this example we will be using the `details` pod. Using `kubectl` get all available upstreams, and the list should
```

Comment:
Why did we pick that one? Bookinfo usually goes to productpage

Reply:
I chose details because it returns json data which is better for a cli example I can change it though